### PR TITLE
Minor improvements in runners + Fix type of `YII_DEBUG` + Add entry points to psalm

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

**Review by thenotsoft — APPROVED** [review]

File: public/index.php [hunk 1]
```diff
@@ -19,15 +20,16 @@
...
 
 /**
- *  Set debug value for web application runner, for default its `true` add additionally the validation of the
- *  container-di configurations (debug mode).
+ * Set debug value for web application runner, for default its `true` add additionally the validation of the
+ * container-di configurations (debug mode).
  */
-define('YII_DEBUG', getenv('YII_DEBUG') ?: true);
+$debug = getenv('YII_DEBUG');
+define('YII_DEBUG', $debug === false || $debug === '' || $debug);
```
Comment (thenotsoft) [inline public/index.php:27]: can simplify
`
define('YII_DEBUG', getenv('YII_DEBUG') ? (bool)getenv('YII_DEBUG') : true);
`

Comment (samdark) [inline public/index.php:27]:
```suggestion
define('YII_DEBUG', getenv('YII_DEBUG') ? (bool)getenv('YII_DEBUG') : true);
```

Comment (samdark) [inline public/index.php:27]:
```suggestion
define('YII_DEBUG', getenv('YII_DEBUG') ? (bool)getenv('YII_DEBUG') : true);
```

File: public/index-test.php [hunk 2]
```diff
@@ -25,15 +26,16 @@
...
 
 /**
- *  Set debug value for web application runner, for default its `true` add additionally the validation of the
- *  container-di configurations (debug mode).
+ * Set debug value for web application runner, for default its `true` add additionally the validation of the
+ * container-di configurations (debug mode).
  */
-define('YII_DEBUG', getenv('YII_DEBUG') ?: true);
+$debug = getenv('YII_DEBUG');
+define('YII_DEBUG', $debug === false || $debug === '' || $debug);
```
Comment (samdark) [inline public/index-test.php:33]:
```suggestion
define('YII_DEBUG', getenv('YII_DEBUG') ? (bool)getenv('YII_DEBUG') : true);
```

File: yii [hunk 3]
```diff
@@ -8,15 +8,16 @@ use App\Runner\ConsoleApplicationRunner;
...
 
 /**
- *  Set debug value for console application runner, for default its `true` add additionally the validation of the
- *  container-di configurations (debug mode).
+ * Set debug value for console application runner, for default its `true` add additionally the validation of the
+ * container-di configurations (debug mode).
  */
-define('YII_DEBUG', getenv('YII_DEBUG') ?: true);
+$debug = getenv('YII_DEBUG');
+define('YII_DEBUG', $debug === false || $debug === '' || $debug);
```
Comment (samdark) [inline yii:15]:
```suggestion
define('YII_DEBUG', getenv('YII_DEBUG') ? (bool)getenv('YII_DEBUG') : true);
```

**Comment by samdark** [comment]
👍

**Pull request merged**